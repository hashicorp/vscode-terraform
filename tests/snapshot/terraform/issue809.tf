locals {
  key_name = "testing"
  test_example = {
    (local.key_name): "test"
  }
}

variable "test" {
  default = "test"
}
