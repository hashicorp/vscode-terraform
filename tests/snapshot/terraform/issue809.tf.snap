>locals {
#^^^^^^ source.terraform meta.block.terraform entity.name.type.terraform
#      ^ source.terraform meta.block.terraform
#       ^ source.terraform meta.block.terraform punctuation.section.block.begin.terraform
>  key_name = "testing"
#^^ source.terraform meta.block.terraform
#  ^^^^^^^^ source.terraform meta.block.terraform variable.declaration.terraform variable.other.readwrite.terraform
#          ^ source.terraform meta.block.terraform variable.declaration.terraform
#           ^ source.terraform meta.block.terraform variable.declaration.terraform keyword.operator.assignment.terraform
#            ^ source.terraform meta.block.terraform variable.declaration.terraform
#             ^ source.terraform meta.block.terraform string.quoted.double.terraform punctuation.definition.string.begin.terraform
#              ^^^^^^^ source.terraform meta.block.terraform string.quoted.double.terraform
#                     ^ source.terraform meta.block.terraform string.quoted.double.terraform punctuation.definition.string.end.terraform
>  test_example = {
#^^ source.terraform meta.block.terraform
#  ^^^^^^^^^^^^ source.terraform meta.block.terraform variable.declaration.terraform variable.other.readwrite.terraform
#              ^ source.terraform meta.block.terraform variable.declaration.terraform
#               ^ source.terraform meta.block.terraform variable.declaration.terraform keyword.operator.assignment.terraform
#                ^ source.terraform meta.block.terraform variable.declaration.terraform
#                 ^ source.terraform meta.block.terraform meta.braces.terraform punctuation.section.braces.begin.terraform
>    (local.key_name): "test"
#^^^^^ source.terraform meta.block.terraform meta.braces.terraform meta.mapping.key.terraform punctuation.section.parens.begin.terraform
#     ^^^^^ source.terraform meta.block.terraform meta.braces.terraform meta.mapping.key.terraform support.constant.terraform
#          ^ source.terraform meta.block.terraform meta.braces.terraform meta.mapping.key.terraform keyword.operator.accessor.terraform
#           ^^^^^^^^ source.terraform meta.block.terraform meta.braces.terraform meta.mapping.key.terraform variable.other.member.terraform
#                   ^ source.terraform meta.block.terraform meta.braces.terraform meta.mapping.key.terraform punctuation.section.parens.end.terraform
#                    ^ source.terraform meta.block.terraform meta.braces.terraform meta.mapping.key.terraform keyword.operator.terraform
#                     ^ source.terraform meta.block.terraform meta.braces.terraform meta.mapping.key.terraform
#                      ^ source.terraform meta.block.terraform meta.braces.terraform string.quoted.double.terraform punctuation.definition.string.begin.terraform
#                       ^^^^ source.terraform meta.block.terraform meta.braces.terraform string.quoted.double.terraform
#                           ^ source.terraform meta.block.terraform meta.braces.terraform string.quoted.double.terraform punctuation.definition.string.end.terraform
>  }
#^^ source.terraform meta.block.terraform meta.braces.terraform
#  ^ source.terraform meta.block.terraform meta.braces.terraform punctuation.section.braces.end.terraform
>}
#^ source.terraform meta.block.terraform punctuation.section.block.end.terraform
>
>variable "test" {
#^^^^^^^^ source.terraform meta.block.terraform entity.name.type.terraform
#        ^ source.terraform meta.block.terraform
#         ^^^^^^ source.terraform meta.block.terraform variable.other.enummember.terraform
#               ^ source.terraform meta.block.terraform
#                ^ source.terraform meta.block.terraform punctuation.section.block.begin.terraform
>  default = "test"
#^^ source.terraform meta.block.terraform
#  ^^^^^^^ source.terraform meta.block.terraform variable.declaration.terraform variable.other.readwrite.terraform
#         ^ source.terraform meta.block.terraform variable.declaration.terraform
#          ^ source.terraform meta.block.terraform variable.declaration.terraform keyword.operator.assignment.terraform
#           ^ source.terraform meta.block.terraform variable.declaration.terraform
#            ^ source.terraform meta.block.terraform string.quoted.double.terraform punctuation.definition.string.begin.terraform
#             ^^^^ source.terraform meta.block.terraform string.quoted.double.terraform
#                 ^ source.terraform meta.block.terraform string.quoted.double.terraform punctuation.definition.string.end.terraform
>}
#^ source.terraform meta.block.terraform punctuation.section.block.end.terraform
>